custom_dc/custom_data.md

```diff
@@ -180,6 +186,16 @@ The first set of parameters only applies to `foo.csv`. The second set of paramet
 
 You must specify the provenance details under `sources`.`provenances`; this field associates one of the provenances defined there to this file.
 
+`observationProperties`
+
+: Optional: Additional information about the observations contained in the CSV file. Currently, four properties are supported:
+- `unit`: The unit of measurement used in the observations. This is a string representing a currency, area, weight, volume, etc. For example, `SquareFoot`, `USD`, `Barrel`, etc.
```

Comment:
Consider linking to the corresponding glossary term for each property in [here](https://docs.datacommons.org/glossary.html#scaling-factor).

Reply:
Good idea. Done.